source/change-streams.rst

```diff
@@ -97,6 +60,10 @@ An error is considered resumable if it meets any of the following criteria:
     * - CursorKilled
       - 237
 
+- a server error response without an error code or with an error code not
+  matching the above criteria, but with an error message containing the
+  substring "not master" or "node is recovering"
+
```

Comment:
I thought *all* other server errors were supposed to be resumable, but this text seems to imply that only "not master" or "node is recovering" server errors are resumable.

Does that mean that all other server errors are NOT resumable?

Reply:
It is separated into errors that contain a "code" and those that don't. Actually, since change streams are only used on 3.6+ I don't think it should be possible to get an error from the server without a "code". Maybe this section is not necessary?

Reply:
This wording is confusing. It's supposed to be an additional condition in which we resume. If there is no error code, or if it's part of the exclusion list, then check the error message. I'll clarify this.